app/views/programs/show.html.erb

```diff
@@ -2,6 +2,8 @@
 
 <h3> <%= @program.genre %> </h3>
 
+<h3> Average rating: <%= @program.average %> </h3>
```

Comment:
Using the `.average` method. You can do `program.reviews.average(:rating)`. Note that this will return `nil` if there are no reviews.